sig-community/charter.md

```diff
@@ -0,0 +1,60 @@
+# SIG Community
+
+## Scope
+
+This SIG cares about all aspects of what is helpful and harmful to this open source software community.
+We understand that this project is a human endeavor, and it is our job to put people before practice, process, and even principle.
```

Comment:
Nit: move this above **Scope** as kind of a mission statement

Reply:
Cool, thanks. I'm going to do this here in a split way so I don't outdate the PR by doing it in my own repo.